src/z_data/z_store.rs

```diff
@@ -168,7 +168,7 @@ impl<F: LurkField> ZStore<F> {
     ) -> (ZExprPtr<F>, ZExpr<F>) {
         let mut expr = ZExpr::RootSym;
         let mut ptr = expr.z_ptr(poseidon_cache);
-        for s in sym.path().iter().rev() {
+        for s in sym.path().iter() {
```

Comment:
This is the fix for the content-addressing of symbols

Reply:
Ah, do you mean the keyword representation? I didn't realize that was being bundled with this work. If so, understood. If otherwise, please clarify.

Reply:
No, the keyword representation was fixed in the PR that introduced packages.

This fix is needed because we are hashing symbols in the wrong order on `master`. We need to mimic what the `Store` does with `Ptr`s. And that `.rev()` was messing up the order.

Reply:
Extracted to another PR